Signal/src/contact/OWSContactsManager.m

```diff
@@ -131,34 +131,70 @@ - (void)handleAddressBookChanged
 
 #pragma mark - Setup
 
-- (void)setupAddressBook {
+- (void)setupAddressBookIfNecessary
+{
     dispatch_async(ADDRESSBOOK_QUEUE, ^{
```

Comment:
Both `[application: didFinishLaunchingWithOptions:]` and `[applicationDidBecomeActive:]` both prod `OWSContactsManager` to update its contacts, so on app launch we do this twice, which doubles our bulk usage of the contacts intersection endpoint.  This logic de-bounces that.